### DISCUSSION TITLE
Since this is essentially torrenting the episode, will I need a VPN if I live in the US?

### DISCUSSION THREAD

**Discussion by Nachtel** [pystardust/ani-cli#238] [Q&A] [answered]
Title says it all

**Comment by port19x** [comment, marked as answer]
No, this does not torrent, and therefore you don't need a VPN.
This tool is essentially a special purpose web browser that connects to gogoanime. 
If you would go on gogoanime without a VPN, like most of us, you can use this tool without a VPN too.
If in the distant future we add a torrent option, then a VPN will be advised and clearly recommended in the readme.

**Reply by Nachtel** [reply]
My bad for misunderstanding the program. Thank you for the clarification.
